### PR TITLE
Open usb device filtered by vendor and product ref #94

### LINKED ISSUE #94: Add check to get wallet by skycoinfoundation
**Describe the bug**
`GetDevice` should check for manufacturer as `SkycoinFoundation` similar to the js lib

use manufacturer or its equivalent field

**Actual behavior**
Check for manufacturer == `SkycoinFoundation` 

**Expected behavior**
return device only if its by `SkycoinFoundation`

**Possible Implementation**
Check how js lib does it

### PR DESCRIPTION
**Pull request by stdevAlDen**
Fixes #94

Changes:
- Filer usb device by [vendor](https://github.com/skycoin/hardware-wallet/blob/develop/tiny-firmware/bootloader/usb.c#L57) and [product](https://github.com/skycoin/hardware-wallet/blob/develop/tiny-firmware/bootloader/usb.c#L58).

- Add a `getRawEntropy` and `getMixedEntropy` to get entropy from the physical device.

Does this change need to mentioned in CHANGELOG.md?
yes

Requires changes in protobuff specs?
no

Requires testing
no

### CODE REVIEW

**Comment by stdevAlDen** [comment]
> Fixes #94
> 
> Changes:
> 
> * Filer usb device by [vendor](https://github.com/skycoin/hardware-wallet/blob/develop/tiny-firmware/bootloader/usb.c#L57) and [product](https://github.com/skycoin/hardware-wallet/blob/develop/tiny-firmware/bootloader/usb.c#L58).
> 
> Does this change need to mentioned in CHANGELOG.md?
> yes
> 
> Requires changes in protobuff specs?
> no
> 
> Requires testing
> no

@olemis & @therealssj [a bit different from js](https://github.com/skycoin/hardware-wallet-js/blob/master/device-wallet.js#L70-L73), but I think it's ok too, please, let me know what do you think?

**Comment by olemis** [comment]
@stdevAlDen pls add a command to output device HID info , inb case it's an USB wallet . I don't think it'll be useful for `EMULATOR` device type so just fail .

**Comment by stdevAlDen** [comment]
> @stdevAlDen pls add a command to output device HID info , inb case it's an USB wallet . I don't think it'll be useful for `EMULATOR` device type so just fail .

@olemis I can create an struct with information from intersection betwen this two structs:
[HidDeviceInfo](https://github.com/skycoin/hardware-wallet-go/blob/master/src/device-wallet/usb/usbhid/hid.go#L81) and [Device_Descriptor](https://github.com/skycoin/hardware-wallet-go/blob/master/src/device-wallet/usb/usbhid/libusb.go#L708)
or make use [this](https://github.com/skycoin/hardware-wallet-go/blob/master/src/device-wallet/usb/bus.go#L22) struct, which option yoy prefer?

File: vendor/github.com/gogo/protobuf/gogoproto/Makefile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Protocol Buffers for Go with Gadgets
```
Comment (olemis) [inline vendor/github.com/gogo/protobuf/gogoproto/Makefile]: Isn't this already vendored in hw-protob  ?

Comment (stdevAlDen) [reply vendor/github.com/gogo/protobuf/gogoproto/Makefile]: @olemis fixed in https://github.com/skycoin/hardware-wallet-go/pull/101/commits/842a62fae3e66fb9ea6c86d3dde301ba6cd79da0

**Pull request merged**